software/robot_utilities.py

```diff
@@ -79,6 +79,18 @@ def LOGSTATE(self, state, error=False):
 def sigmoid_controller(current, target, x_scale = 1, y_scale = 1):
     return (2 / (1 + np.exp(3*(target-current)/x_scale)) - 1) * y_scale
 
+def num_format(number):
+    if number < 10:
+        return "   " + str(number)
+    elif number < 100:
+        return "  " + str(number)
+    elif number < 1000:
+        return " " + str(number)
+    elif number < 10000:
+        return str(number)
+    else:
+        return "wtf from num_format" 
+
```

Comment:
Perhaps you could solve adding the tabulations by making the number into string, counting the amount of 0-s and then add tabulations based on the result?